<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-LNNQE2O" name="travelsTests.MainTest" tests="14" failures="1" timestamp="2019-12-13T17:18:44 CET" time="199.214" errors="0">
  <testcase name="Login" time="18.196" classname="travelsTests.MainTest"/>
  <testcase name="DashboardMenu" time="12.385" classname="travelsTests.MainTest"/>
  <testcase name="UpdatesMenu" time="11.840" classname="travelsTests.MainTest"/>
  <testcase name="ModulesMenu" time="11.312" classname="travelsTests.MainTest"/>
  <testcase name="ToursMenu" time="15.413" classname="travelsTests.MainTest"/>
  <testcase name="CarsMenu" time="15.920" classname="travelsTests.MainTest"/>
  <testcase name="CouponsMenu" time="11.865" classname="travelsTests.MainTest"/>
  <testcase name="NewsletterMenu" time="11.823" classname="travelsTests.MainTest"/>
  <testcase name="BookingsMenu" time="11.245" classname="travelsTests.MainTest"/>
  <testcase name="CarList" time="16.162" classname="travelsTests.MainTest"/>
  <testcase name="CarList10" time="2.113" classname="travelsTests.MainTest"/>
  <testcase name="CarList50" time="3.207" classname="travelsTests.MainTest">
    <failure type="java.lang.AssertionError" message="expected:&lt;true&gt; but was:&lt;false&gt;">
      <![CDATA[java.lang.AssertionError: expected:<true> but was:<false>
at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.failNotEquals(Assert.java:834)
at org.junit.Assert.assertEquals(Assert.java:118)
at org.junit.Assert.assertEquals(Assert.java:144)
at travelsTests.MainTest.CarList50(MainTest.java:174)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CarList50 -->
  <testcase name="AddingCar" time="26.270" classname="travelsTests.MainTest"/>
  <testcase name="Customers" time="31.463" classname="travelsTests.MainTest"/>
</testsuite> <!-- travelsTests.MainTest -->
